Add troubleshooting guide for Service Mesh (#4355)
kyma-project/kyma: docs/service-mesh/14-02-no-access.md

---
title: Can't access Console UI or other endpoints
type: Troubleshooting
---

> A response of code `503` error when trying to access the Console UI or any other endpoint in Kyma can be caused by a configuration error in the Istio Ingress gateway. As a result, the endpoint you call is not exposed.

Reviewer: ```suggestion
The `503` status code received when you try to access the Console UI or any other endpoint in Kyma indicates a configuration error in the Istio Ingress gateway. As a result, the endpoint you call is not exposed.
```

Author: Well, unfortunately, this is not binary. A 503 error CAN be caused by the described gateway error but there are other things that can cause it too. I'll apply your suggestion but I'll edit it a bit to indicate that this is not ALWAY the case,

To fix this problem, restart the Pods of the gateway.

> 1. List all of the available endpoints:

Reviewer: ```suggestion
1. List all available endpoints:
```

  ```
  kubectl get virtualservice --all-namespaces
  ```

> 2. Check all of the ports used by the gateway:

Reviewer: ```suggestion
2. Check all ports used by the gateway:
```
